windows/RNSVG/RenderableView.h

```diff
@@ -87,15 +83,13 @@ struct RenderableView : RenderableViewT<RenderableView> {
   float m_strokeOpacity{1.0f};
   float m_strokeMiterLimit{0.0f};
   float m_strokeDashOffset{0.0f};
+  std::vector<float> m_adjustedStrokeDashArray;
   RNSVG::SVGLength m_strokeWidth{1.0f, RNSVG::LengthType::Pixel};
   Windows::Foundation::Collections::IVector<RNSVG::SVGLength> m_strokeDashArray{
       winrt::single_threaded_vector<RNSVG::SVGLength>()};
-  Microsoft::Graphics::Canvas::Geometry::CanvasCapStyle m_strokeLineCap{
-      Microsoft::Graphics::Canvas::Geometry::CanvasCapStyle::Flat};
-  Microsoft::Graphics::Canvas::Geometry::CanvasLineJoin m_strokeLineJoin{
-      Microsoft::Graphics::Canvas::Geometry::CanvasLineJoin::Miter};
-  Microsoft::Graphics::Canvas::Geometry::CanvasFilledRegionDetermination m_fillRule{
-      Microsoft::Graphics::Canvas::Geometry::CanvasFilledRegionDetermination::Winding};
+  int32_t m_strokeLineCap{0};
+  int32_t m_strokeLineJoin{0};
+  int32_t m_fillRule{1};
```

Comment:
Feels like you want to map these to a constant and not a magic value, especially since they're not all 0. Or at a bare minimum comment what the default is